### PR TITLE
Add php-cs-fixer rules issue #5

### PR DESCRIPTION
**Pull request by germanozambelli**

### CODE REVIEW

**Review by massimo-me — APPROVED** [review]

**Comment by ChiarilloMassimo** [comment]
LGTM

#approved 🚀

**Review by massimo-me — CHANGES_REQUESTED** [review]
Ops

File: .php_cs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...953818,"sample\/get_user.php":1246607117,"sample\/create_user.php":389500120,"sample\/update_refund.php":58664397,"sample\/create_refund.php":2672331567,"sample\/daily_closure.php":3556022720,"sample\/find_users.php":319331321,"sample\/find_charges.php":2035084474,"sample\/get_refund.php":2754399573,"sample\/config.php":3530352292,"sample\/check_auth.php":3900066530,"sample\/get_charge.php":2167637255,"sample\/create_charge.php":1951690050,"src\/Handler\/UserHandler.php":2772911479,"src\/Handler\/AbstractHandler.php":1538838159,"src\/Handler\/ChargeHandler.php":1692978241,"src\/Handler\/DailyClosureHandler.php":3018664356,"src\/Handler\/RefundHandler.php":3286759304,"src\/Handler\/BearerHandler.php":1341661377,"src\/Model\/Charge.php":1601666581,"src\/Model\/ArrayCollection.php":3330407444,"src\/Model\/Refund.php":1465828884,"src\/Model\/User.php":73923446,"src\/Model\/AbstractEntity.php":568911538,"src\/Utils\/PropertyAccess.php":1676438045,"src\/Core\/SatispayConstants.php":2741211552,"src\/Http\/Client.php":3569826686,"src\/Http\/Response.php":1072183878,"src\/Exception\/SatispayException.php":3165153923,"src\/Satispay.php":132358819,"src\/Authorization\/Bearer.php":1355862937}}
```
Comment (massimo-me) [inline .php_cs.cache]: Add in `.gitignore`

File: composer.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
   },
   "require": {
       "php": ">=5.5",
-      "guzzlehttp/guzzle": "^6.3"
+      "guzzlehttp/guzzle": "^6.3",
+    "friendsofphp/php-cs-fixer": "^2.13"
```
Comment (massimo-me) [inline composer.json]: Add two spaces 🙄

**Review by massimo-me — COMMENTED** [review]
We don't have a pre-commit hook, please add in `README.me` the command for running `php-cs-fixer`

File: .php_cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules(
+        [
+            '@Symfony' => true,
```
Comment (massimo-me) [inline .php_cs]: Remove this rule

File: .gitignore [hunk 4]
```diff
@@ -1 +1,7 @@
 vendor/
+
+###> friendsofphp/php-cs-fixer ###
+/.php_cs.cache
+###< friendsofphp/php-cs-fixer ###
+
+.idea/
```
Comment (massimo-me) [inline .gitignore]: Remove `.idea` and add a new line at the end of file

🚨 Add this rule in your `.gitignore` global (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

**Review by massimo-me — CHANGES_REQUESTED** [review]

**Review by massimo-me — APPROVED** [review]

**Pull request merged**